build.gradle.kts

```diff
@@ -459,6 +459,7 @@ tasks.rat {
     "**/licenses/*.txt",
     "**/licenses/*.md",
     "integration-test/**",
+    "docs/*.md",
```

Comment:
I think we have to fix the license header for docs, right? Maybe not in this PR, can you please create an issue to track this?

Reply:
But it also includes like "Copyright 2023 Datastrato Pvt Ltd", is it OK?

Reply:
See #4050

Reply:
I also removed all "Copyright 2023 Datastrato Pvt Ltd". While most of the content is still copyright Datastrato, ASF policy is to mention that in the NOTICE file (known as a relocated copyright statement).